modules/assim.sequential/R/sda.enkf.R

```diff
@@ -485,6 +574,8 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL){
       return(sqrt(diag(x)))})))  #need to make this from quantiles for lyford plot data
     #YCI = YCI[,pmatch(colnames(X), names(obs.mean[[nt]][[1]]))]
    
+    ylab.names<-c("MgC/ha/yr","KgC/m^2","m^2/m^2","gC/m^2","gC/m^2","","","cm")
```

Comment:
ultimately we should grab the units from the database or from the variables argument -- this hardcoded list will break LINKAGES